base/client.jl

```diff
@@ -66,7 +66,15 @@ function repl_cmd(cmd, out)
             end
             cmd = `$shell -c $shell_escape_cmd`
         end
-        run(ignorestatus(cmd))
+        try
+            run(ignorestatus(cmd))
+        catch
+            # strip backtrace and display error
+            # mainly for Windows
```

Comment:
It's a bit unclear to me what "mainly for windows" here refers to.

Reply:
When it isn't on Windows, commands go to a shell that prints an error but doesn't produce a Julia exception. On Windows, there is a Julia exception. But I don't know if there are other cases where this might not hold true.

Reply:
Aha, I didn't know that was the case.